contributing-guidelines.rst

```diff
@@ -46,6 +46,8 @@ Merge Rules
 
 * Every PR should have at least one code review before merging
 * All CI tests should pass
+* Write a one-line merge commit message with the following format so that a changelog can be autogenerated:
+  [Fix/Enhancement]: Change summary
```

Comment:
We should think about that yet. E.g. Packaging and API (I am thinking about different name for the last one) are nice as well. But I would like to keep it simple & short.

Reply:
Yeah, we can definitely extend the options. This PR is more like a discussion-starter.